lib/linguist/languages.yml

```diff
@@ -3355,6 +3355,13 @@ Squirrel:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+Stan:
+  type: programming
```

Comment:
We should choose a color for this language (it will be displayed in the language statistics bar). Is there an official logo from which we could extract a color?